src/frontends/paddle/tests/common.cmake

```diff
@@ -0,0 +1,116 @@
+# Copyright (C) 2018-2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# Test model generating
+set(CODE_ROOT_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../)
+set(PADDLE_REQ "${CODE_ROOT_DIR}/requirements.txt")
+if(Python3_Interpreter_FOUND)
+    execute_process(
+        COMMAND ${Python3_EXECUTABLE} "${CODE_ROOT_DIR}/paddle_pip_check.py" ${PADDLE_REQ}
+        RESULT_VARIABLE EXIT_CODE
+        OUTPUT_VARIABLE OUTPUT_TEXT
+        ERROR_VARIABLE ERROR_TEXT)
+endif()
+
+if(NOT EXIT_CODE EQUAL 0)
+    set(paddlepaddle_FOUND OFF)
+    message(WARNING "Python requirement file ${PADDLE_REQ} is not installed, PaddlePaddle testing models weren't generated, some tests will fail due models not found")
+else()
+    set(paddlepaddle_FOUND ON)
+endif()
+
+ov_add_test_target(
+    NAME ${TARGET_NAME}
+        ROOT ${CODE_ROOT_DIR}
+        DEPENDENCIES
+            paddle_test_models_${PD_MODEL_TAG}
+            openvino_paddle_frontend
+            paddle_fe_standalone_build_test
+        LINK_LIBRARIES
+            openvino::cnpy
+            frontend_shared_test_classes
+            openvino_paddle_frontend
+            openvino::runtime
+            gtest_main_manifest
+            func_test_utils
+        ADD_CLANG_FORMAT
+        LABELS
+            ${ctest_labels} PADDLE_FE
+)
+
+
+if(paddlepaddle_FOUND)
+    execute_process(
+        COMMAND ${Python3_EXECUTABLE} -c "import paddle; print(paddle.__version__)"
+        OUTPUT_VARIABLE PADDLE_VERSION
+        OUTPUT_STRIP_TRAILING_WHITESPACE
+    )
+    message(STATUS "PaddlePaddle version: ${PADDLE_VERSION}")
+    if(PADDLE_VERSION VERSION_GREATER_EQUAL "3.0.0" OR PADDLE_VERSION VERSION_EQUAL "0.0.0")
+        set(PADDLEDET_OPS_URL "https://raw.githubusercontent.com/PaddlePaddle/PaddleDetection/release/2.8.1/ppdet/modeling/ops.py")
+        set(PADDLEDET_OPS_SHA256 "9b3193d91d617a6c9e6ef49896dc8612cbcbb67c146d0a76ce16bf2b64dac86f")
+        set(PADDLEDET_DIRNAME ${CMAKE_CURRENT_BINARY_DIR}/thirdparty/PaddleDetection/release281/ppdet/modeling/)
+        set(paddle_gen_tag "ge3")
```

Comment:
The tag `ge3` and `ge2` mean the two PDPD model generation paths.
- `ge3` means the PDPD>=3.0.0 model gen path, which includes `.pdmodel` and `.json`(defualt) paths.
- `ge2` means the PDPD<3.0.0 model gen path, which only includes legacy `.pdmodel` path.
My understanding is right?